### DISCUSSION TITLE
Provide support of Scala 3

### DISCUSSION THREAD

**Discussion by prabutdr** [codewars/codewars.com#3188] [Ideas]
Please provide support of submitting solutions in scala 3

**Comment by hobovsky** [comment]
Languages, packages, and updates of those  should be requested on the [issue board of the `codewars/runner` repository](https://github.com/codewars/runner/issues).

Scala 3 is already supported by Codewars: https://docs.codewars.com/languages/scala but there is a [list of kata](https://github.com/codewars/content-issues/wiki/List-of-Scala-Kata-to-Update) which does not work with Scala 3 and has to be updated by users. You are welcome to help :)
